docarray/array/storage/weaviate/backend.py

```diff
@@ -71,9 +72,15 @@ def _init_storage(
 
         self._persist = bool(config.name)
 
-        self._client = weaviate.Client(
-            f'{config.protocol}://{config.host}:{config.port}'
-        )
+        if config.timeout_config:
+            self._client = weaviate.Client(
+                f'{config.protocol}://{config.host}:{config.port}',
+                timeout_config=config.timeout_config,
+            )
+        else:
+            self._client = weaviate.Client(
+                f'{config.protocol}://{config.host}:{config.port}',
+            )
         self._config = config
```

Comment:
here we have 2 options:
* give our own default parameters (with the burden of maintaining values in case client changes)
* follow the pattern here: https://github.com/jina-ai/docarray/blob/main/docarray/array/storage/annlite/backend.py#L70

@JoanFM  what's your opinion ?

Reply:
I like the one where less attention from us is needed

Reply:
done